resources/css/app.css

```
@@ -438,3 +438,55 @@
     background: #e2e8f0;
     border-radius: 9999px;
 }
+
+.prose-ticket p { margin: 0 0 0.75rem 0; }
+.prose-ticket p:last-child { margin-bottom: 0; }
+.prose-ticket a { color: #5B619D; text-decoration: underline; word-break: break-word; }
```

Stylelint (17.9.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)